client/src/redux/donation-saga.js

```diff
@@ -55,6 +59,26 @@ function* addDonationSaga({ payload }) {
   }
 }
 
+function* createStripeSessionSaga({ payload: { stripe, data } }) {
+  try {
+    const session = yield call(postCreateStripeSession, {
+      ...data,
+      location: window.location.href
+    });
+    stripe.redirectToCheckout({
+      sessionId: session.data.id
+    });
+  } catch (error) {
+    const data =
+      error.response && error.response.data
+        ? error.response.data
+        : {
+            message: defaultDonationError
+          };
+    yield put(addDonationError(data.message));
```

Comment:
```suggestion
    const err =
      error.response && error.response.data
        ? error.response.data.message
        : defaultDonationError;
    yield put(addDonationError(err));
```
Saves a couple of lines.